packages/teams-js/test/public/media.spec.ts

```diff
@@ -838,7 +713,146 @@ describe('media', () => {
           },
         } as DOMMessageEvent);
 
-        return expect(promise).rejects.toEqual({ errorCode: ErrorCode.SIZE_EXCEEDED });
+        await expect(promise).rejects.toMatchObject({ errorCode: ErrorCode.SIZE_EXCEEDED });
```

Comment:
Based on the unit test guidelines Erin wrote up yesterday, we prefer to use toEqual instead of toMatchObject for validating SdkError objects.

"toMatchObject  will only check the object's properties that are passed to the toMatchObject function, so in general, prefer greater specificity with toEqual"